.lychee.toml

```diff
@@ -25,3 +21,7 @@ exclude = [
     "https://github.com/.*/(pull|issues)/[0-9]+", # We have too many PR and issues links in CHANGELOG.md that we allways run out of the rate limit
     "https://bugs.mysql.com/*", # Failing with 403 Forbidden. Likely rejecting the user agent
 ]
+max_concurrency = 20
+max_retries = 5
```

Comment:
This fixes a typo, `max_retires` -> `max_retries`. [Reference](https://github.com/lycheeverse/lychee/blob/a071bb4e2880afdc16dd072ac9a30724ce560352/lychee.example.toml#L32)